arangod/Agency/Node.h

```diff
@@ -334,15 +344,18 @@ class Node final {
   //  unit tests updated.
   //
   /// @brief Get node specified by path string
-  Node& getOrCreate(std::string const& path);
+  Node& getOrCreate(std::string_view path);
 
   /// @brief Get node specified by path string
   std::optional<std::reference_wrapper<Node const>> get(
-      std::string const& path) const;
+      std::string_view path) const;
 
   /// @brief Get string value (throws if type NODE or if conversion fails)
   std::optional<std::string> getString() const;
 
+  /// @brief Get string value (throws if type NODE or if conversion fails)
+  std::optional<std::string_view> getStringView() const;
```

Comment:
Maybe you can return null string as nullopt?

Reply:
I prefer to stick to the convention used in this file.

Reply:
Ok